e2e/project/frontity.settings.ts

```diff
@@ -46,6 +50,10 @@ const settings: Settings<
         name: "@frontity/tiny-router",
         state: { router: { autoFetch: false } },
       },
+      {
+        name: "@frontity/wp-source",
+        state: { source: { url: "https://test.frontity.org/" } },
+      },
```

Comment:
Because `tiny-router` has to access `state.source.get()` in: https://github.com/frontity/frontity/blob/6db0054df1a27813eb8888dd06e401480dccc5d2/packages/tiny-router/src/actions.ts#L52 we need to add both tiny-router and wp-source.